src/panels/config/integrations/integration-panels/ozw/ozw-config-network.ts

```diff
@@ -46,203 +29,61 @@ class OZWConfigNetwork extends LitElement {
-                    </div>
-                    <div class="secondary">
-                      ${this.hass.localize(
-                        "ui.panel.config.ozw.common.ozw_instance"
-                      )}
-                      ${this._network.ozw_instance}
-                      ${this._statistics
-                        ? html`
-                            &bull;
-                            ${this.hass.localize(
-                              "ui.panel.config.ozw.network.node_count",
-                              "count",
-                              this._statistics.node_count
-                            )}
-                          `
-                        : ``}
-                      <br />
-                      ${this.hass.localize(
-                        "ui.panel.config.ozw.common.controller"
-                      )}:
-                      ${this._network.getControllerPath}<br />
-                      OZWDaemon ${this._network.OZWDaemon_Version} (OpenZWave
-                      ${this._network.OpenZWave_Version})
-                    </div>
-                  </div>
-                  <div class="card-actions">
-                    ${this._generateServiceButton("add_node")}
-                    ${this._generateServiceButton("remove_node")}
-                  </div>
-                </ha-card>
-              `
-            : ``}
-        </ha-config-section>
-      </hass-tabs-subpage>
+      </ozw-network-router>
     `;
   }
 
   static get styles(): CSSResultArray {
-    return [
-      haStyle,
-      css`
-        .secondary {
-          color: var(--secondary-text-color);
-        }
-        .online {
-          color: green;
-        }
-        .starting {
-          color: orange;
-        }
-        .offline {
-          color: red;
-        }
-        .content {
-          margin-top: 24px;
-        }
-
-        .sectionHeader {
-          position: relative;
-          padding-right: 40px;
-        }
-
-        .network-status {
-          text-align: center;
-        }
-
-        .network-status div.details {
-          font-size: 1.5rem;
-          margin-bottom: 16px;
-        }
-
-        .network-status ha-svg-icon {
-          display: block;
-          margin: 0px auto 16px;
-          width: 48px;
-          height: 48px;
-        }
-
-        .network-status small {
-          font-size: 1rem;
-        }
-
-        ha-card {
-          margin: 0 auto;
-          max-width: 600px;
-        }
-
-        .card-actions.warning ha-call-service-button {
-          color: var(--error-color);
-        }
-
-        .toggle-help-icon {
-          position: absolute;
-          top: -6px;
-          right: 0;
-          color: var(--primary-color);
-        }
-
-        ha-service-description {
-          display: block;
-          color: grey;
-          padding: 0 8px 12px;
-        }
-
-        [hidden] {
-          display: none;
-        }
-      `,
-    ];
+    return [haStyle];
```

Comment:
```suggestion
return haStyle;
```

Reply:
Github really messed with this suggestion. Basically no need to have the array here